bigbluebutton-html5/client/main.jsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
+import htmlBBB from '/imports/api/bbb';
```

Comment:
If use for initialization, should be inside `imports/startup/client`, makes hard to understand, and the eslinter won't accept as a correct statement, will show an [no unused var](http://eslint.org/docs/rules/no-unused-vars) error.